bom/pom.xml

```diff
@@ -216,18 +216,18 @@
 
       <dependency>
         <groupId>com.redhat.service.smartevents</groupId>
-        <artifactId>manager-integration-tests</artifactId>
```

Comment:
Can you please remove the BOM entries from here too?

Reply:
Looks already removed to me..